packages/utils/__tests__/vue/global-node.test.ts

```diff
@@ -37,8 +37,18 @@ describe('global-nodes', () => {
 
     expect(el.parentElement).toBe(document.body)
 
+    changeGlobalNodesTarget(target)
     changeGlobalNodesTarget(target)
```

Comment:
Why are we calling it twice?

Reply:
Sorry, I rechecked the code, this seems redundant. I have deleted.